addons/sms/wizard/sms_composer.py

```diff
@@ -83,6 +84,14 @@ def _compute_composition_mode(self):
                 else:
                     composer.composition_mode = 'comment'
 
+    @api.depends('res_model')
+    def _compute_res_model_description(self):
+        for composer in self:
```

Comment:
Maybe we could batch the easiest part, like
```
self.res_model_description = False
for composer in self.filtered('res_model'):
    composer.res_model_description = self.env['ir.model']._get(composer.res_model).display_name
```

Reply:
Hello @tde-banana-odoo 

if we don't have res_model then `res_model_description` will be  unassigned and it will raise the error.
for better we can use inline condition.

Thanks

Reply:
Hence the first part: first assign False (self.res_model_description = False), then update only relevant :) .